### PR TITLE
Sb couchdb3.3.3

### PR DESCRIPTION
**Pull request by SteffenBrinckmann**
CouchDB 3.3.3 (which is now automatically pushed via snap) has other installation requirements
Attempt to adopt to those requirements

### CODE REVIEW

**Comment by SteffenBrinckmann** [comment]
After discussing on https://github.com/apache/couchdb/issues/4945

# identify issues
snap info couchdb
ls -l /snap/couchdb/current/snap/hooks/configure 
tail -n 20 /var/log/syslog

# add cookie
cd /var/snap/couchdb/current/etc
echo $(dd if=/dev/random bs=1 count=38 status=none | base64 | tr -cd '[:alnum:]')
sudo vi vm.args
-setcookie UNX4319uTTWLyhyEl6WC0H2v492WFO5kGZAYzsh8cj1WIBvMU8

# add configure hooks
cd /var/snap/couchdb/current/etc
sudo vi local.ini
[couchdb]
database_dir = /var/snap/couchdb/common/data
view_index_dir = /var/snap/couchdb/common/data


# restart and test
sudo snap restart couchdb
curl http://127.0.0.1:5984/

Might have to be removed in a future version

**Comment by SteffenBrinckmann** [comment]
# add cookie
grep setcookie /var/snap/couchdb/current/etc/vm.args
echo $(dd if=/dev/random bs=1 count=38 status=none | base64 | tr -cd '[:alnum:]')
sudo sed -i -e 's/-setcookie monster/-setcookie UNX4319uTTWLyhyEl6WC0H2v492WFO5kGZAYzsh8cj1WIBvMU8/g' /var/snap/couchdb/current/etc/vm.args

# add configure hooks
grep database_dir /var/snap/couchdb/current/etc/local.ini
sudo sed -i -e 's/\[couchdb\]/\[couchdb\]\ndatabase_dir = \/var\/snap\/couchdb\/common\/data\nview_index_dir = \/var\/snap\/couchdb\/common\/data/g' /var/snap/couchdb/current/etc/local.ini

**Comment by SteffenBrinckmann** [comment]
@jmurugan-fzj The upstream tool (couchdb) has updated the version and now we need to adopt the installation script
Only one change is required: installationTools.py (the other changes are from the previous PR)
Please review.

**Comment by jmurugan-fzj** [comment]
@SteffenBrinckmann I still see some error in the db installation, is this expected? "process-control" plug is missing, will it create any problem?
image

File: pasta_eln/installationTools.py [hunk 1]
```diff
@@ -184,9 +184,10 @@ def installLinuxRoot(couchDBExists:bool, pathPasta:Path=Path(''), password:str='
       conf = createDefaultConfiguration('admin', password, pathPasta)
       with open(path,'w', encoding='utf-8') as fConf:
         fConf.write(json.dumps(conf, indent=2) )
+    cookie = ''.join(random.choice(string.ascii_letters+string.digits) for _ in range(42))
```
Comment (jmurugan-fzj) [inline pasta_eln/installationTools.py:187]: If it's some random string, why cannot we just just use uuid?
https://docs.python.org/3/library/uuid.html

**Comment by SteffenBrinckmann** [comment]
@jmurugan-fzj I did those changes:
- random strings are created via uuid
- process-control is not initiated anymore

**Review by jmurugan-fzj — APPROVED** [review]

**Pull request merged**